fuzz/fuzz_targets/fuzz_glob.rs

```diff
@@ -0,0 +1,25 @@
+#![no_main]
+
+extern crate globset;
+extern crate libfuzzer_sys;
+
+use globset::Glob;
+use libfuzzer_sys::fuzz_target;
+use std::str::FromStr;
```

Comment:
This should be written like this:

```rust
use std::str::FromStr;

use {globset::Glob, libfuzzer_sys::fuzz_target};
```

That is, imports should be grouped by: 1) things from std, 2) things from third party dependencies and 3) (none in this case) crate-internal imports.